src/quo2/components/qr_code/qr.cljs

```diff
@@ -0,0 +1,32 @@
+(ns quo2.components.qr-code.qr
+  (:require [react-native.core :as rn]))
+
+(defn user-profile-qr-code
+  [{:keys [key-uid public-key port qr-size]}]
+  (let [profile-qr-url         (str "https://join.status.im/u/" public-key)
```

Comment:
better to move this outside of quo, quo shouldn't have any logic, or knowledge about status app, `"https://join.status.im/u/"` should be in config or constants

Reply:
hmm does it make sense to move this component entirely to `src/status_im2/common/qr`?

Reply:
yes

Reply:
resolved here : https://github.com/status-im/status-mobile/pull/15528/commits/6dd9897540ca2ab37b0d190dfae3cd80ddb1b56d